pkg/api/api.go

```diff
@@ -10,13 +10,34 @@ type ErrorResp struct {
 // BlockVolumeCreateReq represents gluster block
 // volume create request
 type BlockVolumeCreateReq struct {
-	HaCount            int      `json:"ha"`
-	AuthEnabled        bool     `json:"auth-enabled"`
-	FullPrealloc       bool     `json:"full-prealloc"`
-	Size               uint64   `json:"size"`
-	Storage            string   `json:"storage"`
-	RingBufferSizeInMB uint64   `json:"ring-buffer-size-mb"`
-	Hosts              []string `json:"hosts"`
+	// Name represents block Volume name
+	Name               string      `json:"name,omitempty"`
+	HostingVolume      string      `json:"hosting_volume,omitempty"`
```

Comment:
This is not decided yet, whether the glusterd blockd rest server will take care of handing block hosting volume creation and retrieval.

Reply:
gluster-block cli requires hosting volume name also for block vol creation

Reply:
then its should not be omitempty